prow/private_deck.md

```diff
@@ -0,0 +1,188 @@
+# Setting up Private Deck
+
+## 1) Set up TenantIDs for ProwJobs and Repos
```

Comment:
To make this more actionable, would recommend mentioning whether this step should be done by prow user or prow operator